examples/windows/windows-ps-diskperf.yml

```diff
@@ -0,0 +1,11 @@
+# Windows Disk Performance metrics from Win32_PerfFormattedData_PerfDisk_LogicalDisk
+# Requires PowerShell v3+ because of the use of Get-CimInstance
+integrations:
+  - name: nri-flex
+    config:
+      name: winDiskPerf
+      apis:
+        - name: winDiskPerf
+          shell: powershell
+          commands:
+            - run: "Get-CimInstance -ClassName Win32_PerfFormattedData_PerfDisk_LogicalDisk | Select-Object -Property Name,AvgDiskBytesPerRead,AvgDiskBytesPerTransfer,AvgDiskBytesPerWrite,AvgDiskQueueLength,AvgDiskReadQueueLength,AvgDisksecPerRead,AvgDisksecPerTransfer,AvgDisksecPerWrite,AvgDiskWriteQueueLength,CurrentDiskQueueLength,DiskBytesPersec,DiskReadBytesPersec,DiskReadsPersec,DiskTransfersPersec,DiskWriteBytesPersec,DiskWritesPersec,FreeMegabytes,PercentDiskReadTime,PercentDiskTime,PercentDiskWriteTime,PercentFreeSpace,PercentIdleTime,SplitIOPerSec | ConvertTo-Json"
```

Comment:
@jbiggley - can you edit this into a multi-line yaml for readability?

```yaml
- run: |
     Get-CimInstance...
     next line of text...
```

Reply:
Updated multi-line layout for readability and tested to verify functionality.